memories/memory.go

```diff
@@ -0,0 +1,29 @@
+package memory
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo"
+)
+
+func CreateMemory(ctx echo.Context) error {
+	return ctx.String(http.StatusOK, "successfully create memory")
+}
+
+func EditMemory(ctx echo.Context) error {
```

Comment:
CRUDに沿うならここはUpdateのほうがよさそうね
```suggestion
func UpdateMemory(ctx echo.Context) error {
```